### DISCUSSION TITLE
Way to organize repos

### DISCUSSION THREAD

**Discussion by Rusinas** [community/community#17151] [Other Feature Feedback, Questions, & Ideas]
If you have more than 10 repos, managing them becomes painful. We really need a way to organize them: tags, folders (d&d is welcome!), various view options. Last one is essential IMO. I have 2k display and even if I using my browser fullscreen, I can only see maximum of 9 repos in the list. Not to mention that I usually don't use it fullscreen. 

P.S. Current "Lists" feature doesn't provide any help. I don't even understand what it's for.

**Comment by radar-blip** [comment]
I like the folders idea. That would probably be the best bet.
